src/pages/Campaign/CampaignItem.tsx

```diff
@@ -122,6 +123,12 @@ const CampaignItem = ({ campaign, onSelectCampaign, isSelected }: CampaignItemPr
   const isShowProgressBar = isOngoing && account && campaign?.userInfo?.status === CampaignUserInfoStatus.Eligible
   const percentVolume = !tradingVolumeRequired ? 0 : (tradingVolume / tradingVolumeRequired) * 100
   const percentTradingNumber = !tradingNumberRequired ? 0 : (tradingNumber / tradingNumberRequired) * 100
+  const isPassedVolume = percentVolume >= 100
+  const isPassedNumberOfTrade = percentTradingNumber >= 100
+  const isQualified =
```

Comment:
Look complicated??! Just >= tradingVolumeRequired and >= tradingNumberRequired?

Reply:
my condition make sure that: if a campaign `not required tradingVolume` or `not required tradingNumber`. it will always correct.